web/tests/Web.E2ETests/Features/School/CompareYourCosts.feature

```diff
@@ -41,7 +41,7 @@ Feature: School compare your costs
           | Test academy school 162 | West Northamptonshire   | Academy special converter          | 231              | £1,597,953  |
           | Test academy school 465 | Enfield                 | Academy 16-19 converter            | 231              | £1,597,953  |
           | Test school 102         | Hammersmith and Fulham  | Community school                   | 212              | £1,587,223  |
-          | Test school 154         | Kirklees                | Community school                   | 174              | £1,545,387  |
+          | Test school 154         | Test Local Authority    | Community school                   | 174              | £1,545,387  |
```

Comment:
Are these changes intended?

Reply:
Yes - see comment in 'Checklist' and PR #1686.

Reply:
The same thing will happen with the upcoming PR for Platform dependencies due to the underlying test data in #1686 (and possibly elsewhere) having already been applied to `d02`.